.github/workflows/integration-tests.yml

```
@@ -146,7 +146,10 @@ jobs:
           touch .env
           echo 'C8Y_BASEURL="${{ secrets.C8Y_BASEURL }}"' >> .env
           echo 'C8Y_USER="${{ secrets.C8Y_USER }}"' >> .env
+          echo 'C8Y_TENANT="${{ secrets.C8Y_TENANT }}"' >> .env
```

What's the purpose of this secret? Was this not implied by `C8Y_BASEURL`?

---

I think adding ID to the name helps to understand what it is more. At least I get confused which one, tenant name (the one in URL) or tenant ID (tXXXX)

```suggestion
          echo 'C8Y_TENANT_ID="${{ secrets.C8Y_TENANT_ID }}"' >> .env
```

---

Though `C8Y_TENANT` is common place in a lot of tooling, go-c8y-cli for one ;)

---

For the qaenvironment it is still one of the older tenant which it's tenant id does not confirm to the `txxxxx`...as this standard was only introduced at a later point in c8y's history